x-pack/platform/plugins/shared/triggers_actions_ui/public/common/alert_rule_from_vis_ui_action/alert_flyout_component.tsx

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
Should be `rule_flyout_component`

"An alert" is something a rule generates. It's ok to call it "alert rule" or "alerting rule" but the singular is always a "rule"

Reply:
Fixed via ac0b19c